apps/ledger-live-mobile/src/components/RootNavigator/ExchangeLiveAppNavigator.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useMemo } from "react";
+import { createStackNavigator } from "@react-navigation/stack";
+import { useTheme } from "styled-components/native";
+import { Icons, Flex } from "@ledgerhq/native-ui";
+import { useFeature } from "@ledgerhq/live-common/featureFlags/index";
+import { ScreenName } from "../../const";
+import { getStackNavigatorConfig } from "../../navigation/navigatorConfig";
+
+import PlatformApp from "../../screens/Platform/App";
+import styles from "../../navigation/styles";
+
+export default function ExchangeLiveAppNavigator({ route }: any) {
+  const { colors } = useTheme();
+
+  // PTX smart routing feature flag - buy sell live app flag
+  const ptxSmartRouting = useFeature("ptxSmartRouting");
+
+  const stackNavigationConfig = useMemo(
+    () => getStackNavigatorConfig(colors, true),
+    [colors],
+  );
+
+  const { params: routeParams } = route;
+
+  return (
+    <Stack.Navigator {...stackNavigationConfig}>
+      <Stack.Screen
+        name={ScreenName.ExchangeBuy}
+        options={{
+          headerBackImage: () => (
+            <Flex pl="16px">
+              <Icons.CloseMedium color="neutral.c100" size="20px" />
+            </Flex>
+          ),
+          headerStyle: styles.headerNoShadow,
+          headerTitle: () => null,
+        }}
+      >
+        {(_props: any) => (
+          <PlatformApp
+            {..._props}
+            {...routeParams}
+            route={{
+              ..._props.route,
+              params: {
+                ..._props.route.params,
+                platform: ptxSmartRouting?.params?.liveAppId || "multibuy",
+                mode: "buy",
+                currency: _props.route.params?.defaultCurrencyId,
+                account: _props.route.params?.defaultAccountId,
```

Comment:
We can remove the duplication of data by doing that
```suggestion
                ..._props.route.params,
                platform: ptxSmartRouting?.params?.liveAppId || "multibuy",
                mode: "buy",
                currency: _props.route.params?.defaultCurrencyId,
                account: _props.route.params?.defaultAccountId,
                defaultCurrencyId: undefined,
                defaultAccountId: undefined,
```